Changelog.md

```diff
@@ -11,6 +11,7 @@ Breaking Changes:
 * Remove `include_chain_clauses_in_custom_matcher_descriptions` option
   and make it the default. (Phil Pirozhkov, #1279)
 * Remove support for present-tense dynamic predicate. (Phil Pirozhkov, #1286)
+* Restrict block matchers use with value expectations. (Phil Pirozhkov, #1285)
```

Comment:
I'd mention the implicit blocks part, hopefully this PR will help people understand why we want to do this :joy:
```suggestion
* Prevent implicit blocks (e.g blocks as values) from being used with block matchers.
  (Phil Pirozhkov, #1285)
```

Reply:
I hope so as well! :smile: